cpp/include/raft/stats/cov.cuh

```diff
@@ -89,7 +90,8 @@ void cov(const raft::handle_t& handle,
   static_assert(
     std::is_same_v<LayoutPolicy, raft::row_major> || std::is_same_v<LayoutPolicy, raft::col_major>,
     "Data layout not supported");
-  RAFT_EXPECTS(data.size() == covar.size(), "Size mismatch");
+  RAFT_EXPECTS(data.extent(1) == covar.extent(0) && data.extent(1) == covar.extent(1),
```

Comment:
`data.extents() == covar.extents()` works too!

Reply:
Not for this one because covar is `ncols*ncols` while data is `nrows*ncols`

Reply:
Oh oof, sorry!  Yes, for next time : - )